### PR TITLE
Testing and Requirement Duplicates

### PR DESCRIPTION
**Pull request by aeslaughter**
Adds unittests for the newly created SQA extension commands
Adds ability to check for duplicate requirements (it is off by default now, but once we get it cleaned up it will be on by default)

### CODE REVIEW

**Comment by aeslaughter** [comment]
```bash
 ./sqa_check.py --duplicates
```

This turned up 108 requirements that have one or more duplicates!

File: framework/doc/content/source/actions/AdaptivityAction.md [hunk 1]
```diff
@@ -1,14 +1 @@
...
-The AdaptivityAction has not been documented. The content contained on this page includes the
-typical automatic documentation associated with an Action; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
-
-!syntax description /Executioner/Adaptivity/AdaptivityAction
-
-!syntax parameters /Executioner/Adaptivity/AdaptivityAction
-
-!bibtex bibliography
+!template load file=stubs/moose_action.md.template name=AdaptivityAction syntax=/Executioner/Adaptivity/AdaptivityAction
```
Comment (dschwen) [inline framework/doc/content/source/actions/AdaptivityAction.md]: This adds another step to turning an undocumented object into a documented object :-(. Previously one could just start from the stub, but now that structure is gone.

Comment (aeslaughter) [reply framework/doc/content/source/actions/AdaptivityAction.md]: I know this involves running the server and copy and pasting, but does this help alleviate the problem you have with this?

My idea is that you generate, serve, and go to the page. You see the following, which you copy and paste and start working. Is that no good?

https://mooseframework.inl.gov/docs/PRs/13692/site/source/auxkernels/MaterialRealTensorValueAux.html

**Comment by aeslaughter** [comment]
I am closing this in favor of #13692, which contains all these changes as well.

**Review by permcody — APPROVED** [review]

**Pull request merged**